test/src/common/harnessIo.h

```diff
@@ -0,0 +1,16 @@
+/***********************************************************************************************************************************
+Harness for Testing
```

Comment:
Another comment that should be more descriptive.

Reply:
Updating to Harness for Io Testing would be in line with the others.